docs/user-guide/attributes-table.md

```diff
@@ -92,7 +96,7 @@ In the Attribute table it is possible to apply two types of filter (better expla
 
 * [Quick filter](filtering-layers.md#quick-filter)
```

Comment:
In the Attribute table it is possible to apply filters in three ways (as explained in the [Filtering layers](filtering-layers.md) section):

Add here a third point to link also the **click on map** filter option.